docs/plugins/github-apps.md

```diff
@@ -0,0 +1,78 @@
+# Using GithubApps for backend authentication
```

Comment:
```suggestion
# Using GitHub Apps for Backend Authentication
```

Reply:
Just noting that this file doesn't have any yaml at the start and isn't exposed. I'll presume that that's intentional.

Reply:
It's intentional for now until we can dogfood the API in the scaffolder